docs/interfaces.md

```diff
@@ -463,6 +463,13 @@ trusted apps.
 
 * Auto-Connect: no
 
+### timedate-control
+
+Can modify time/date exposed via D-Bus by systemd-timedated and read all
+all properties of /org/freedesktop/timedate1 D-Bus object.
+
+* Auto-Connect: yes
+
 ### timeserver-control
 
 Can manage timeservers directly separate from ``config core``.
```

Comment:
Can you update this to be:
```
Can manage timeservers via systemd-timedated and directly separate from ``config core``

### timezone-control

Can manage timezone via systemd-timedated and directly separate from ``config core``

* Auto-Connect: no
```

In other words, update timeserver-control to reference systemd-timedated and add timezone-control.